src/libs/testing-library/utils/routing.ts

```diff
@@ -1,25 +1,26 @@
 import {
-  encode,
   RootRoute,
   Route,
   Router,
   createMemoryHistory,
 } from '@tanstack/react-router';
 import { RouterRenderParams } from './types';
-import { debugTokens } from '../../../e2e/utils/types';
+import { parseSearchWith } from 'libs/routing/utils';
+import { isAddress } from 'ethers/lib/utils';
 
-export const tokens = [
-  {
-    address: debugTokens.ETH,
-    decimals: 18,
-    symbol: 'ETH',
-  },
-  {
-    address: debugTokens.USDC,
-    decimals: 6,
-    symbol: 'USDC',
-  },
-];
+const encodeValue = (value: string | number | symbol) => {
+  if (typeof value == 'string' && isAddress(value)) return `${String(value)}`;
```

Comment:
Why wrap the value with `String` ? the back quote should do it no ?

Reply:
That's right - removed